docs/corpora_schema.md

```diff
@@ -96,17 +109,21 @@ development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:000039
 
 The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
```

Comment:
From my earlier notes:

[**Editorial**: *Inline this sentence in the table. Keep requirements together. This is a SHOULD?*]

Reply:
Thanks for catching this. I inlined it, but missed deleting the original sentence.